src/sql/platform/jobManagement/browser/jobActions.ts

```diff
@@ -32,7 +34,7 @@ export enum JobActions {
 export class IJobActionInfo {
 	ownerUri?: string;
 	targetObject?: any;
-	component: JobManagementView;
+	component: JobManagementView | NotebookHistoryComponent;
```

Comment:
Does the `NotebookHistoryComponent` not extend the `JobManagementView` class?

Reply:
It does. I'll remove that.